dnf-behave-tests/features/module-install-globs.feature

```diff
@@ -0,0 +1,59 @@
+Feature: Installing module profiles with globs
+
+
+Background:
+  Given I use the repository "dnf-ci-fedora-modular"
+    And I use the repository "dnf-ci-fedora"
+
+
+Scenario: Install a module profile with glob in name
+   When I execute dnf with args "module install node*"
+   Then the exit code is 0
+    And Transaction contains
+        | Action                    | Package                                       |
+        | module-stream-enable      | nodejs:8                                      |
+        | module-profile-install    | nodejs/default                                |
+        | install                   | nodejs-1:8.11.4-1.module_2030+42747d40.x86_64 |
+
+
+Scenario: Install a module profile with glob in name and stream
+   When I execute dnf with args "module install node*:*0"
+   Then the exit code is 0
+    And Transaction contains
+        | Action                    | Package                                           |
+        | module-stream-enable      | nodejs:10                                         |
+        | module-profile-install    | nodejs/default                                    |
+        | install                   | nodejs-1:10.11.0-1.module_2200+adbac02b.x86_64    |
+
+
+Scenario: Install a module profile with glob in version
+  Given I use the repository "dnf-ci-fedora-modular-updates"
+   When I execute dnf with args "module install nodejs:10:20180920*"
+   # by selecting version, the older RPM from dnf-ci-fedora-modular gets installed
```

Comment:
Looks like a copy-paste leftover.

Reply:
Could you be more specific?
If you mean the comment, it's there by purpose - to give a hint how the test is performed

Reply:
Sorry, I was too fast, misread the comment... I thought the comment is talking about the actual RPM software, but it is talking about an RPM package. Apparently, you're correct in your nomenclature :grin: quoting wikipedia:

> The name RPM refers to the following: the .rpm file format, files in the .rpm file format, software packaged in such files, and the package manager program itself.

Doesn't really help though :slightly_smiling_face:

Reply:
Understand. I changed the text slightly to make it less confusing.